draft-ietf-oauth-identity-assertion-authz-grant.md

```diff
@@ -337,22 +308,22 @@ The example below uses an ID Token as the Identity Assertion, and uses a JWT Bea
     &audience=https://acme.chat.example/
     &resource=https://api.chat.example/
     &scope=chat.read+chat.history
-    &subject_token=eyJraWQiOiJzMTZ0cVNtODhwREo4VGZCXzdrSEtQ...
-    &subject_token_type=urn:ietf:params:oauth:token-type:id_token
+    &subject_token=7SliwCQP1brGdjBtsaMnXo...
+    &subject_token_type=urn:ietf:params:oauth:token-type:access_token
     &client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
     &client_assertion=eyJhbGciOiJSUzI1NiIsImtpZCI6IjIyIn0...
 
 ### Processing Rules
```

Comment:
We will need to refresh the processing rules for an Access Token.  We currently don't prescribe it must be a JWT or whether is must conform to the JWT Access Token profile.  

The ID-JAG may contain authentication  claims as it previously was exchanged from an ID Token.  Authentication context now must travel via the Access Token.

Reply:
I agree, a JWT format access token should be prescribed (but not required).

Reply:
I'd say a strong reference to RFC9068 would be useful but by no means would I want to require that.

Reply:
reference. I don't want to force anyone into a particular access token format.